src/ports/postgres/modules/lda/test/lda.sql_in

```diff
@@ -474,3 +474,89 @@ select assert(array_upper(perplexity_iters,1)  = 2, 'Perplexity iterations are d
 select assert(perplexity[1] > 0 , 'Perplexity value should be greate than 0') from lda_model ;
 select assert(array_upper(ARRAY(Select distinct unnest(perplexity)),1)= array_upper(perplexity,1) , 'Perplexity values should be unique') from lda_model ;
 
+
+-- Test for evaluate_every = 1  and 0 : In this case the iterations should not stop early --
```

Comment:
@hpandeycodeit 
I can't find the test for evaluate_every = 0. Am i missing something ?

Reply:
when evaluate_every = NULL (it takes the default evaluate_every=0) and in that case, we don't calculate perplexity. We have a test case for covering evaluate_every = NULL.